src/OpenTelemetry/SuppressInstrumentationScope.cs

```diff
@@ -15,13 +15,15 @@
 // </copyright>
 
 using System;
+using System.Diagnostics;
 using OpenTelemetry.Context;
 
 namespace OpenTelemetry
 {
     public sealed class SuppressInstrumentationScope : IDisposable
     {
         private static readonly RuntimeContextSlot<bool> Slot = RuntimeContext.RegisterSlot<bool>("otel.suppress_instrumentation");
+        private static readonly RuntimeContextSlot<ActivitySpanId?> ActivitySlot = RuntimeContext.RegisterSlot<ActivitySpanId?>("otel.suppress_instrumentation_activity");
```

Comment:
can we reuse the same Slot instead of creating new one?

Reply:
Reiley had the same question, just answered above.